railties/lib/rails/application/finisher.rb

```diff
@@ -10,10 +10,6 @@ class Application
     module Finisher
       include Initializable
 
-      initializer :add_generator_templates do
-        config.generators.templates.unshift(*paths["lib/templates"].existent)
-      end
```

Comment:
I've moved this to [`Application#load_generators`](https://github.com/rails/rails/pull/45544/files#diff-84292fcaae327f1a70e7c3b1ebb55b193dea016c1118a8566930bc5cd2a51ba5R496-R499) because `rails credentials:edit` does not run initializers since #34789.  (Otherwise `lib/templates/rails/credentials` would not be [included in the template paths](https://github.com/rails/rails/blob/cd08e6bcc4cd8948fe01e0be1ea0c7ca60373a25/railties/lib/rails/generators.rb#L70) of `CredentialsGenerator`.)

Reply:
Do we have to worry about users that may have defined an initializer `before: :add_generator_templates`?